core/store/src/lib.rs

```diff
@@ -574,6 +574,12 @@ impl StoreUpdate {
         self.transaction.merge(other.transaction)
     }
 
+    #[tracing::instrument(
+        level = "debug",
+        target = "store",
+        "StoreUpdate::commit",
+        skip_all,
+    )]
```

Comment:
Unfortunately I do not think we will be able to have these be at the debug level. These events happen thousands of times every minute (a good question to figure out: why?) which is likely to make it intractable to have these "on by default" from the cost perspective. The best option for us would be to have this at the `trace` level and then modify `target` so that these individual spans can be enabled on a case-by-case basis.

If you can set these all just to the `trace` level for now, I'm happy to improve on the targets as a followup.